modules/heat_transfer/doc/content/modules/heat_transfer/sqa/heat_transfer_far.md

```diff
@@ -0,0 +1 @@
+!template load file=sqa/module_far.md.template category=heat_transfer module=Heat Conduction
```

Comment:
Maybe you are missing "Heat Conduction" -> "Heat Transfer"?

Reply:
good catch

Reply:
fixed